wagtail/wagtailforms/models.py

```diff
@@ -295,7 +295,15 @@ def process_form_submission(self, form):
 
     def send_mail(self, form):
         addresses = [x.strip() for x in self.to_address.split(',')]
-        content = '\n'.join([x[1].label + ': ' + text_type(form.data.get(x[0])) for x in form.fields.items()])
+        content = []
+        for field in form:
+            value = field.value()
+            if isinstance(value, list):
+                value = ', '.join(value)
+                if len(value) == 0:
```

Comment:
Shouldn't this be dedented one level so an empty string value is handled the same as an empty list value? And more importantly, is this even required behaviour? 

As a Python developer I would confuse this with an actual None. As an end user I would confuse this with someone actually entering (or selecting) the value None in a form field.